packages/api/src/feature_flag/queries/create_feature_flag.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO feature_flag (
+  name,
+  description,
+  globally_enabled,
+  created_at,
+  updated_at
```

Comment:
No need to include these fields in the INSERT statement; the database's default values will be the ones we want

Reply:
Ideally we would, but `promo` is an older table that was created without default values for anything other than the ID, so we'd have to update the database first.

Reply:
Ah, I understand, ok.